srcpkgs/sqlite-replication/template

```diff
@@ -29,15 +28,21 @@ pre_configure() {
 	sed -i -e 's/ -ltinfo//g' configure
 }
 
+post_install() {
+	vmkdir usr/lib/pkgconfig
+	mv "$DESTDIR"/usr/lib{/sqlite-replication,}/pkgconfig/sqlite3.pc
+	# Alpine just deletes the binary, but someone may want it for debugging
+	mv "$DESTDIR"/usr/bin/sqlite3{,.sqlite-replication}
```

Comment:
Imo this should be `vmove`d to a separate package.

Reply:
why though?

Reply:
It's unnecessary for normal use of the package, and you say keep it for debugging; hence why I personally think it should be in a separate package.

Reply:
made the change